src/actions/assetsActions.js

```diff
@@ -650,7 +643,7 @@ export const resetAccountAssetsBalancesAction = (accountId: string) => {
   };
 };
 
-export const fetchAllAccountsAssetsBalancesAction = () => {
+export const fetchAllAccountsAssetsBalancesAction = (isRefresh?: boolean) => {
```

Comment:
Same as above

Reply:
New name is isRefreshingPart. changes done.